### PR TITLE
Implement Vet Center designs

### PR DESCRIPTION
**Pull request by bshyong**
tagging https://github.com/department-of-veterans-affairs/vets-api/pull/1025 for awareness

### CODE REVIEW

File: src/js/facility-locator/components/ServicesAtFacility.jsx [hunk 1]
```diff
@@ -117,10 +68,12 @@ class ServicesAtFacility extends Component {
   }
 
   renderVetCenterServices() {
-    // to be hardcoded
     return (
       <div className="mb2">
         <ul>
+          {vetCenterServices.map(s => {
+            return <li key={s}>{s}</li>;
```
Comment (U-DON) [inline src/js/facility-locator/components/ServicesAtFacility.jsx]: Is a key separated by spaces valid?

File: src/js/facility-locator/components/FacilityMap.jsx [hunk 2]
```diff
@@ -17,6 +17,7 @@ class HowToGetHere extends Component {
       va_health_facility: 'health',
       va_cemetery: 'cemetery',
       va_benefits_facility: 'benefits',
+      vet_center: 'vet-centers',
```
Comment (U-DON) [inline src/js/facility-locator/components/FacilityMap.jsx]: Would it be easier to maintain this by transforming the values of `facilityTypes` from `config` using `_.kebabCase`?

**Review by U-DON — APPROVED** [review]
LGTM.

File: src/js/facility-locator/components/FacilityMap.jsx (hunk 2 above)
Comment (bshyong) [inline src/js/facility-locator/components/FacilityMap.jsx]: Yes, good catch. I'd like to do some clean up on this once both the Access to Care and Vet Center features are deployed..

The switch to camelCase I implemented in the Access to Care branch so once that gets in I can also implement your suggestion

**Pull request merged**